priv/static/.gitignore

```diff
@@ -1,2 +0,0 @@
-/*
```

Comment:
i vaguely remember we had issues when we didn't have this .gitignore in here. but i can't remember what it was. i assume you worked out that we no longer need this here?

Reply:
I've undone this. Mistakenly ran `make clean` somewhere and this got removed